example/lib/main.dart

```diff
@@ -55,7 +55,7 @@ const html = """
 <body>
   <main>
     <div id="icon">&#x1F3C7;</div>
-    <div id="text">Press the button below to sign in using your Localtest.me account.</div>
+    <div id="text">Press the button below to sign in using your Localhost account.</div>
```

Comment:
I think that this is better suited for a separat PR so that we can update the screenshots, and make sure that this works on all platforms...

```suggestion
    <div id="text">Press the button below to sign in using your Localtest.me account.</div>
```